install/kubernetes/helm/istio/charts/galley/values.yaml

```diff
@@ -0,0 +1,6 @@
+#
+# galley configuration
+#
+enabled: true
```

Comment:
I prefer these component level enabled flags to be in global, so that users know what they are enabling. Otherwise, people will pile up a lot of obscure options in these local values.yaml and fill it with inconsistent stuff. Creates a bigger mess.

Reply:
it can be overwritten from the parent's values.yaml..  that is the best part!

Reply:
@rshriram agreed - I think we should specify a default in values.yaml for every subchart with its name as well as the key/value pair enabled: true/false.  The lowest friction approach is to then duplicate this to the subcharts.